tests/integration_tests/test_sim.py

```diff
@@ -52,11 +52,13 @@ def test_arming(self, qtbot, sim_app, device_type):
         flush_packets(sim_app, device_type)
 
         assert sim_app.rocket_data.last_value_by_device(device_type, DataEntryIds.STATE) == 1
+        assert sim_app.rocket_data.last_value_by_device(device_type, DataEntryIds.EVENT) == DataEntryValues.EVENT_ARMED
```

Comment:
I jumped the gun it seems; the SIM arming test (test_arming) is actually failing on this assertion. All other tests pass.